index.js

```diff
@@ -1 +1,124 @@
 // implement your API here
+
+const express = require('express');
+const server = express();
+
+const Users = require('./data/db');
+
+server.use(express.json());
+
+server.get('/', (request, response) => {
+  response.send('Hello from our new server!!!');
+});
+
+//POST
+
+server.post('/api/users', (request, response) => {
+  if (!request.body.name && !request.body.bio) {
+    response.status(400).json({
+      success: false,
+      error: 'Please provide name and bio for the user.'
+    });
+  } else {
+    Users.insert(newUser)
+      .then(user => {
+        response.status(201).json({ success: true, user });
+      })
+      .catch(err => {
+        response.status(500).json({
+          success: false,
+          error: 'There was an error while saving the user to the database'
+        });
+      });
+  }
+});
+
+//GET
+server.get('/api/users', (request, response) => {
+  Users.find()
+    .then(users => {
+      response.send(users);
```

Comment:
I would do `response.status(200).json(users)`, but I think this would work - it just wouldn't send a status code.